tools/lsp/properties.rs

```diff
@@ -37,18 +38,20 @@ pub(crate) struct PropertyInformation {
 pub(crate) struct ElementInformation {
     id: String,
     type_name: String,
+    range: Option<lsp_types::Range>,
```

Comment:
I think `None` will serialise as `{ ..., range: undefined }` (or `null`) while the `range?: LspRange` seems to make the field optional, not the value. Did you mean to use `#[serde(skip_serializing_if = "Option::is_none")]` perhaps?